entgql/extension.go

```diff
@@ -215,7 +225,7 @@ func (e *Extension) genSchemaHook() gen.Hook {
 // in the template list and returns its index.
 func (e *Extension) whereExists() (int, bool) {
 	for i := range e.templates {
-		if e.templates[i] == WhereTemplate {
+		if e.templates[i].Name() == "gql_where_input" {
```

Comment:
```suggestion
		if e.templates[i].Name() == WhereTemplate.Templates()[1].Name() {
```

Use the parsed one instead of using a raw string here. The result will be the same.